README.md

```diff
@@ -1 +1,3 @@
 This readme is open for contribution
+
+Really?
```

Comment:
^_^

Reply:
;-)

Fyi: My idea was to trying out coping the build results from fork to a PR. I can do it with a github app, but didn't find a very easy way to stop the building until the build on fork is passed. 

For example here the build seems to be green, but in-fact the steps are skipped (that was the plan). I can add more check lines based on the build results from the fork, but still I am not happy with the usability. I would prefer to show somehow that the full build is not executed here.

(Not sure it was clear, it's a tricky scenario and just an experiment...)